atlasdb-feign/src/main/java/com/palantir/atlasdb/http/InterruptHonoringRetryer.java

```diff
@@ -1,45 +0,0 @@
-/*
- * (c) Copyright 2018 Palantir Technologies Inc. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.palantir.atlasdb.http;
-
-import java.io.InterruptedIOException;
-
-import feign.RetryableException;
-import feign.Retryer;
-
-/**
- * Simple wrapper around the default Feign retryer that propagates {@link InterruptedIOException}s
```

Comment:
do we necessarily want to get rid of the interrupt honouring thing? assuming this was to cover missing okhttp capabilities - it's probably worth checking whether or not it has been fixed upstream, otherwise we *might* want to bring it back.

In my haste to get things working quickly, I didn't wrap things